repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/gatherinfocontract/GatherTestInformation.java

```diff
@@ -1,7 +1,8 @@
-package fr.inria.spirals.repairnator.process.step;
+package fr.inria.spirals.repairnator.process.step.gatherinfocontract;
```

Comment:
Ok to change but I'd go more for sthg like `gatherinfo`: it's the step so should be an action, not the result.